File: Python/List-2/sum67.py
def sum67(nums):
  if len(nums) == 0:
    return 0
  result = 0
  for i in range(len(nums)):
    if nums[i] == 6:
      j = 0
      while nums[i+j] != 7:
        nums[i+j] = 0
        j+=1
      nums[i+j] = 0
    result += nums[i]
  return result

File: Java/Array-2/sum67.java
public int sum67(int[] nums) {
  int added = 0;
  int occurenceOf7 = 0;
  for(int i = 0; i < nums.length; i ++){
    if(nums[i] == 6){
      for(int j = i; nums[j] != 7; j++){
        nums[j] =0;
        occurenceOf7 = j;
      }
      nums[occurenceOf7 + 1] = 0;
    }
    else{ 
      added += nums[i];
    }
  }
  return added;
}
